README.md

```diff
@@ -28,6 +28,10 @@ Shall the action fail if a license violation is detected? Default `false`
 
 **Required** Path to the config file for the license checker.
 
+### `generate-dash`
```

Comment:
As a reader its not clear how `dash` and `clearlydefined` work together. Please provide a better explaination how these 2 are connected.